admin_manual/configuration/server/occ_command.rst

```diff
@@ -1797,33 +1833,64 @@ Listing Users
 ^^^^^^^^^^^^^
 
 You can list existing users with the ``user:list`` command.
-The syntax is::
+The syntax is
+
+.. code-block:: console
 
   user:list [options] [<search-pattern>]
 
 User IDs containing the ``search-pattern`` string are listed. Matching is 
 not case-sensitive. If you do not provide a search-pattern then all users 
 are listed.
 
-This example lists user IDs containing the string ron:: 
+Options:
+
+::
+
+  --output[=OUTPUT]	         Output format (plain, json or json-pretty, default is plain)
+  -a, --attributes[=ATTRIBUTES]  Adds more details to the output
+
+Allowed attributes, multiple values possible
+
+::
+
+  uid, displayName, email, quota, enabled, lastLogin, home, 
+  backend, cloudId, searchTerms [default: ["displayName"]]
+
+This example lists user IDs containing the string ron
```

Comment:
s/containing the string ron/containing the string "ron"./g

Reply:
I suspect that the search string needs to be >=4 chars long these days to actually work, because of the new default of 4 chars before matching names are looked for.

Reply:
correct, changed ron to aron